src/connection.rs

```diff
@@ -5,24 +5,9 @@ use std::rc::Rc;
 
 use mio::tcp::TcpStream;
 use mio::Timeout as TimeoutHandle;
-use mio::{
```

Comment:
So you don't like the collapse? I wonder if that's something I can change about rustfmt